src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -120,6 +126,19 @@
         </div>
     {% endif %}
 
+    {% if collector.firewall|default([]) is not empty %}
+        <h2>Security Firewall</h2>
+
+        <div class="metrics">
+            <div class="metric">
+                <span class="value">{{ collector.firewall.name|default('unknown') }}</span>
+                <span class="label">Firewall Name</span>
```

Comment:
You can use [`profiler`dump`](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php#L34)